### ISSUE TITLE
Exhibit Images: SiteImprove ADA Issues

### ISSUE THREAD

**Issue by kzadrozny** [uchicago-library/library_website#460]
Multiple automated ADA issues with Solo / Duo image stream fields.

## No alt text
- Getting error of no alt text even when `figcaption` is filled in.
- Multiple way to move forward on this. One is to pull a unique ID for each solo / duo div and add a `aria-described-by` tag when caption is filled.
- May also need to simplify `figcaption` section and remove citation area into its own div. (More research needed.)

## Link without a text alternative
- All exhibits for solo / duo images are erroring out for no alt text. Switch from having caption required to having alt text required

**Comment by kzadrozny** [comment]
Alt text issue was a false positive: if caption is provided, no alt text is needed.
SiteImprove seems to have discovered this error as this issue is no longer coming up for the exhibit images.
